dashboard/app/reporting_test.go

```diff
@@ -721,3 +721,62 @@ func TestAltTitles7(t *testing.T) {
 	c.expectEQ(rep.Title, crash1.Title)
 	c.expectEQ(rep.Log, crash2.Log)
 }
+
+func TestDetachExternalTracker(t *testing.T) {
+	c := NewCtx(t)
+	defer c.Close()
+
+	build := testBuild(1)
+	c.client.UploadBuild(build)
+
+	crash1 := &dashapi.Crash{
+		BuildID:     "build1",
+		Title:       "title1",
+		Maintainers: []string{`"Foo Bar" <foo@bar.com>`, `bar@foo.com`},
+		Log:         []byte("log1"),
+		Report:      []byte("report1"),
+		MachineInfo: []byte("machine info 1"),
+	}
+	c.client.ReportCrash(crash1)
+
+	// Get single report for "test" type.
+	resp, _ := c.client.ReportingPollBugs("test")
+	c.expectEQ(len(resp.Reports), 1)
+	rep := resp.Reports[0]
+	c.expectNE(rep.ID, "")
+	c.expectEQ(string(rep.Config), `{"Index":1}`)
+
+	// Signal detach_reporting for current bug.
+	reply, _ := c.client.ReportingUpdate(&dashapi.BugUpdate{
+		ID:         rep.ID,
+		Status:     dashapi.BugStatusUpstream,
+		ReproLevel: dashapi.ReproLevelNone,
+		Link:       "http://URI/1",
+		CrashID:    rep.CrashID,
+	})
+	c.expectEQ(reply.OK, true)
+
+	// Now add syz repro to check it doesn't use first reporting.
+	crash1.ReproOpts = []byte("some opts")
+	crash1.ReproSyz = []byte("getpid()")
+	c.client.ReportCrash(crash1)
+
+	// Fetch bug and check reporting path (Config) is different.
+	resp, _ = c.client.ReportingPollBugs("test")
```

Comment:
Here you can do just:
```
rep2 := c.client.pollBug()
```
since we don't need any special handling (replying with BugStatusUpstream).